docs/etl/insert_examples.py

```diff
@@ -0,0 +1,12 @@
+from pathlib import Path
```

Comment:
This is such a cool way of building docs!

![](https://c.tenor.com/md3foOULKGIAAAAC/magic.gif)